pkg/services/store/entity/README.md

```diff
@@ -137,4 +137,14 @@ GF_GRAFANA_APISERVER_STORAGE_TYPE=unified-grpc ./bin/grafana server
 You can then list the previously-created playlists with:
 ```sh
 kubectl --kubeconfig=./grafana.kubeconfig get playlist
-```
\ No newline at end of file
+```
+
+#### Changing protobuf interface
```

Comment:
I think it's better to be separate section: not under `Running Unified Storage`
```suggestion
## Changing protobuf interface
```

Reply:
Thanks for paying attention!